LibreNMS/OS/CienaSds.php

```diff
@@ -158,37 +158,40 @@ public function discoverEntityPhysical(): Collection
         }));
 
         // Interface stuff
-        $interfaceIndexMapping = SnmpQuery::walk('BRIDGE-MIB::dot1dBasePortIfIndex')->table(1);
+        $interfaceIndexMapping = SnmpQuery::walk('IF-MIB::ifIndex')->table(1);
```

Comment:
This walk is not useful, you can just delete it.  it is always going to be 1 => 1, 2 => 2, 3 => 3, etc.